dxp/how_clay_arrives_to_dxp.md

```diff
@@ -1,9 +1,158 @@
 # How Clay arrives to DXP?
+
 The goal of this section is to explain:
-- How is it brought into DXP?
-- How is it imported into theme?
-- How to import it into a custom-theme?
-- How is it imported into a module?
-- How to import it into a custom-module?
-- Which options do we have?
-- What can we use of Clay once imported? (and why)
\ No newline at end of file
+
+-   How is it brought into DXP?
+-   How is it imported into theme?
+-   How to import it into a custom-theme?
+-   How is it imported into a module?
+-   How to import it into a custom-module?
+-   What can we use of Clay once imported? (and why)
+-   Which options do we have?
+
+## How is Clay brought into DXP?
+
+There are 2 main methods to import Clay into DXP:
+
+-   as a Theme Inherence
+-   as a NPM Dependency
```

Comment:
NPM → npm

(After years of writing it "NPM", @julien and @izaera convinced me the other day not to do this anymore... because npm is such a special company, and it has a special name that transcends the rules and conventions of the English language... 🤦‍♂️)